README.md

```diff
@@ -47,6 +47,8 @@ Runtime developers can build runtime implementations that run OCI-compliant bund
 
 There is a loose [Road Map](./ROADMAP.md).
 During the `0.x` series of OCI releases we make no backwards compatibility guarantees and intend to break the schema during this series.
+Current and previous releases of the spec in HTML and PDF formats can be found in [Releases](./releases).
+Release notes summarizing key changes are also included here.
```

Comment:
On Tue, Apr 26, 2016 at 05:27:56PM -0700, Rob Dolin (MSFT) wrote:
> > +Current and previous releases of the spec in HTML and PDF formats can be found in [Releases](./releases).
> > +Release notes summarizing key changes are also included here.
>
> @wking Thanks.  I was thinking "also included here" referred to the
> previous location hyperlinked in the prior sentence.

In that case I'd use “there”, but as I mentioned briefly in [1], I
like the repo-local link for the ChangeLog more than a releases link.
It's distributed with the repo, so it's under obvious version control
and the link works even when you or GitHub are offline.  Do you have
reasons for preferring the releases link to a ChangeLog link?

[1]: https://github.com/opencontainers/runtime-spec/pull/383/files/1fa3741d673086cf35d399ecc40492b79c39d57c#r59437515